source/sync/configure/pause-or-terminate-sync.txt

```diff
@@ -139,32 +378,138 @@ terminate and re-enable Device Sync under a few different circumstances:
 
 .. include:: /includes/warn-terminate-sync-client-reset.rst
 
-Procedure
-~~~~~~~~~
+.. tabs::
+
+   .. tab:: UI
+      :tabid: ui
+
+      Terminate Device Sync in your App to stop syncing data across
+      devices. 
+
+      .. procedure::
+
+         .. step:: Press the Terminate Sync Button
+
+            In the :guilabel:`Sync` tab of the App Services UI, click the red
+            :guilabel:`Terminate Sync` button.
```

Comment:
Not that it's terribly confusing without it, but for consistency, should this have an accompanying image?

Reply:
I think I'd rather just remove all of the images, since it has been problematic to keep the UI up to date. I'll try a version without them and see what we think.